### DISCUSSION TITLE
Our Oomph setup catalog does not show up in newer Oomph versions

### DISCUSSION THREAD

**Discussion by phermsdorf** [eclipse-oomph/oomph#104] [General]
Hi,
since a longer time we updated from oopmh 1.29 to the current 1.34 version and noticed that our product setup catalog does not show up after starting the installer.

We append something like the following to the oomph .ini:
`-Doomph.redirection.setups=http://git.eclipse.org/c/oomph/org.eclipse.oomph.git/plain/setups/->http://<hostname>/custom-setup/`

It seems to me like the entry does not get populated to the catalog list, since i can't find that entry when using the "Switch Catalog Index" Menu or the "Catalog Index Manager" Dialog.

There i see some older (local test) catalogs which still seem to work when selected, but the one provided with the property in the .ini file does not show up at all.

Also public available setup catalogs like this one does not seem to work for me (with the given property settings on that site): https://github.com/ArloL/eclipse-projects

Is there anything to adjust with newer oomph versions or what could be the problem?

Thanks for any hints or ideas!

Bye Peter

**Comment by merks** [comment]
I'm not sure how well "legacy" redirections work since we moved to github.

https://github.com/eclipse-oomph/oomph/blob/04645e699be868b45efb1ad474df2b44644abb42/plugins/org.eclipse.oomph.setup.core/src/org/eclipse/oomph/setup/internal/core/SetupContext.java#L141-L143

Best to use `https://raw.githubusercontent.com/eclipse-oomph/oomph/master/setups/` this as the base for redirection.

**Comment by phermsdorf** [comment]
:+1:  :1st_place_medal: 
Great, it works!

Ed, thanks a lot!

**Reply by merks** [reply]
Woo hoo.  I'm glad to be of help.  🥳
